### DISCUSSION TITLE
Roadmap update

### DISCUSSION THREAD

**Discussion by geiregjo** [primefaces/community#2494] [PrimeVue]
Our team decided to go with PrimeVue because it has some interesting things coming. I've been keeping an eye on the roadmap for a while but there's not much movement? Stuff from Q2 is not finished yet and most things for Q3 are not started? Or am I interpreting the roadmap incorrect? We are getting a little worried about to be honest.

Looking forward to an update!

**Comment by NaturesRadar** [comment]
![image](https://github.com/user-attachments/assets/8a1a20fd-fe35-4128-9861-dd4164f0a2a4)
